docs/InstallGuide.md

```diff
@@ -15,23 +15,14 @@ from the root directory of the cloned boss-manage.git repository.*
 You will need a machine installed with the following software packages:
 * Python 3.7 or later
```

Comment:
Tim, I am thinking we should make the default be 3.8.  Maybe we talk about it in todays standup.  
https://aws.amazon.com/blogs/developer/python-support-policy-updates-for-aws-sdks-and-tools/
EOL for 3.7 is December.

Reply:
We can merge whenever you want Tim.

Reply:
I went ahead and merged it.